src/generateTasks.js

```diff
@@ -2,16 +2,19 @@
 
 const minimatch = require('minimatch')
 
+const readConfigOption = require('./readConfigOption')
+
 module.exports = function generateTasks(config, files) {
     const linters = config.linters !== undefined ? config.linters : config
     const resolve = file => files[file]
     return Object.keys(linters)
         .map((pattern) => {
             const commands = linters[pattern]
-            const filter = minimatch.filter(pattern, {
+            const globOptions = readConfigOption(config, 'globOptions', {})
+            const filter = minimatch.filter(pattern, Object.assign({
                 matchBase: true,
```

Comment:
Nit-picking here but can't those defaults go into the 3rd argument at the L13? This should the whole `Object.assign` here